hnn_core/gui/gui.py

```diff
@@ -39,232 +59,428 @@
     "cividis",
 ]
 
-_plot_options = {
-    'Horizontal': 'L-R',
-    'Vertical': 'U-D',
+layer_specific_dipole_viz_conf = {
+    "kwargs": 'sharex=True, gridspec_kw={"height_ratios": [1, 1, 2]}',
+    "plots": {
+        0: [
+            {
+                "sim_name": "",
+                "plot_type": "dipole@L2",
+                "config": {
+                    "max_spectral_frequency": 100,
+                    "dipole_scaling": 3000,
+                    "dipole_smooth": 30,
+                    "spectrogram_cm": _spectrogram_color_maps[0]
+                }
+            },
+        ],
+        1: [
+            {
+                "sim_name": "",
+                "plot_type": "dipole@L5",
+                "config": {
+                    "max_spectral_frequency": 100,
+                    "dipole_scaling": 3000,
+                    "dipole_smooth": 30,
+                    "spectrogram_cm": _spectrogram_color_maps[0]
+                }
+            },
+        ],
+        2: [
+            {
+                "sim_name": "",
+                "plot_type": "dipole@agg",
+                "config": {
+                    "max_spectral_frequency": 100,
+                    "dipole_scaling": 3000,
+                    "dipole_smooth": 30,
+                    "spectrogram_cm": _spectrogram_color_maps[0]
+                }
+            },
+        ]
+    }
+}
+
+
+_default_vis_plot_trees = {
+    "2,1": {
+        "kwargs": 'sharex=True, gridspec_kw={"height_ratios": [1, 3]}',
```

Comment:
I think `sharex='col'` is a better default. If for example you make a 2x2 grid, you don't want to share across all of them but just share the columns

Reply:
> I think `sharex='col'` is a better default. If for example you make a 2x2 grid, you don't want to share across all of them but just share the columns

agreed